src/HtmlUtils.tsx

```diff
@@ -339,6 +339,7 @@ class HtmlHighlighter extends BaseHighlighter<string> {
     }
 }
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
a bit concerning if TS thinks this class isn't used.

Reply:
![image](https://user-images.githubusercontent.com/2403652/91824149-38948980-ec32-11ea-9fc6-90e80170a0b9.png)
![image](https://user-images.githubusercontent.com/2403652/91824155-3b8f7a00-ec32-11ea-939c-aaa213743170.png)

Well it isn't

Reply:
then we can delete it rather than de-lint it?